external/jni/Android-tensorflow-lite-2.3.0.mk

```diff
@@ -89,6 +89,10 @@ CORE_CC_ALL_SRCS += \
 
 TF_LITE_FLAGS += -DEGL_EGLEXT_PROTOTYPES -DCL_TARGET_OPENCL_VERSION=220
 
+# XNNPACK delegate
```

Comment:
recommend to move this to line 54 for the readability :)

Reply:
cool!